src/ttsim/plot_dag.py

```diff
@@ -27,10 +27,14 @@
 
 
 def plot_tt_dag(
-    with_params: bool, inputs_for_main: dict[str, Any], title: str, output_path: Path
+    with_params: bool,
+    inputs_for_main: dict[str, Any],
+    title: str,
+    output_path: Path,
 ) -> None:
     """Plot the taxes & transfers DAG, with or without parameters."""
-
+    if "backend" not in inputs_for_main:
```

Comment:
Do we need this? I think the same check is already in `main`.

Reply:
Probably not, this part of the code was really just a hack to get some visualisation out. As @MImmesberger is working on a proper implementation in #964, we can ignore it here.